services/edus/src/main/java/app/coronawarn/datadonation/services/edus/otp/OtpControllerExceptionHandler.java

```diff
@@ -13,17 +13,17 @@
 @ControllerAdvice
 public class OtpControllerExceptionHandler extends ResponseEntityExceptionHandler {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(OtpControllerExceptionHandler.class);
+  private static final Logger OTP_LOGGER = LoggerFactory.getLogger(OtpControllerExceptionHandler.class);
```

Comment:
What is the difference here? Why was renaming necesary?

Reply:
Because it extends the ResponseEntityExceptionHandler, which has also a Logger LOGGER.Is to prevent missunderstanding.